jaxrs-clients/src/main/java/com/palantir/remoting1/jaxrs/FeignJaxRsClientBuilder.java

```diff
@@ -96,7 +96,7 @@
                     .build(),
                     ConnectionSpec.CLEARTEXT);
 
-    private static final ObjectMapper OBJECT_MAPPER = ObjectMappers.guavaJdk7();
+    private static final ObjectMapper OBJECT_MAPPER = ObjectMappers.guavaJdk8();
```

Comment:
we explicitly prevent user changes to the objectmapper since this would have the potential to make APIs non-interoperable